demo/app.js

```diff
@@ -98,20 +98,24 @@ app.listen(5005)
 				.then(text => fs.writeFileSync(path.join(process.cwd(), 'test-page.html'), text))
 				.then(() => app.close())
 				.then(() => {
+					const files = [
+						'test-page.html',
+						'public/main.css',
+						'public/main.js',
+						'public/main.js.es6.js'
+					];
+					[
+						'o-errors',
+						'vendor',
+						'font-loader'
+					].forEach(bundle => {
+						files.push(
+							`public/n-ui/${bundle}.js`,
+							`public/n-ui/${bundle}.js.es6.js`
```

Comment:
should this be `public/n-ui/${bundle}.es6.js` instead of having 2 `.js`s

Reply:
it looks like they're being built as `${name}.es6.js`, which i think would mean this is right ?